lib/migrate/resolve/resolve.js

```diff
@@ -16,6 +25,15 @@ module.exports = function transformer(j, ast) {
 		return modules.length > 0 && modules[0];
 	};
 
+	/**
+	 *
+	 * Add a `modules` property to the `resolve` object or update the existing one
+	 * based on what is already in `resolve.root`
+	 *
+	 * @param {NodePath} p — wrapped ast property node that represents the `resolve` property
```

Comment:
Node and elaboration in returns

Reply:
Not sure what you mean here by “elaboration in returns”?

Reply:
Elaborate what p returns a bit more 👍

Reply:
I added a description to the `@returns` tag. Do you want me to add descriptions for all of them? In cases where it seemed self-explanatory I omitted the description.